app/controllers/welcome_controller.rb

```diff
@@ -1,3 +1,9 @@
 class WelcomeController < ApplicationController
-  def index; end
+  def show
+    @image = Image.find(params[:id])
+  end
+
+  def index
```

Comment:
Since these actions are both (show and index) related to images, showing and indexing images respectively, let's add the actions (and the view) to ImagesController instead of WelcomeController. From there, we can change the root route and get rid of WelcomeController, WelcomeControllerTest, welcome/index.html.erb, etc. in a separate commit.